tests/indicators/s-z/StarcBands/StarcBands.Tests.cs

```diff
@@ -66,13 +66,13 @@ public void BadData()
     public void NoQuotes()
     {
         List<StarcBandsResult> r0 = noquotes
```

Comment:
To get these tests to pass, you'll have to put them in as 20 here, which is okay for the tests since any number will do.

Reply:
Nevermind.  You'd already caught on to these "no quotes" scenarios well.  Any number will do.